examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -536,6 +558,16 @@ def __getitem__(self, index):
         return example
 
 
+def model_has_vae(args):
```

Comment:
Ok for me! Nicely solved. Guess in the future we could also have a `--vae_pretrained_model_name_or_path`